### PR TITLE
[PE-4456] Added Titled example to Drawer page

### PR DESCRIPTION
**Pull request by janibolkvadze**
Added Drawer with title examples (Web component + HTML) to Drawer component page

@dani-cl-madrid @mattnickles @jllr 

https://ctl.atlassian.net/browse/PE-4456

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://614b124fbfd75a2f5eb2e1d3e33a79e2d79fc733-chi-pr.enterprise.ctl.io" target="_blank"> https://614b124fbfd75a2f5eb2e1d3e33a79e2d79fc733-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/700/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://34a6fefc0c6c4b4f9b9acedb11a77d824a3653e9-chi-pr.enterprise.ctl.io" target="_blank"> https://34a6fefc0c6c4b4f9b9acedb11a77d824a3653e9-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/703/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://c1bb65f41ff79dfb8cef9f4cb2f67a378b7166d3-chi-pr.enterprise.ctl.io" target="_blank"> https://c1bb65f41ff79dfb8cef9f4cb2f67a378b7166d3-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/705/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://a62d837be28fec03c293b58b1b88cc8352ee36de-chi-pr.enterprise.ctl.io" target="_blank"> https://a62d837be28fec03c293b58b1b88cc8352ee36de-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/717/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://ed4f59e209efd223ce0b7313a6ee929126d9a882-chi-pr.enterprise.ctl.io" target="_blank"> https://ed4f59e209efd223ce0b7313a6ee929126d9a882-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/739/ for more details.

File: src/custom-elements/src/components/drawer/drawer.tsx [hunk 1]
```diff
@@ -178,13 +180,32 @@ export class Drawer {
...
+    const target = this.el;
+    const config = {
+      attributes: true,
+      attributeOldValue: true,
+      attributeFilter: ['title']
+    };
+
+    const subscriberCallback = (mutations) => {
+      mutations.forEach((mutation) => {
+        this.drawerTitle = mutation.target.title;
```
Comment (dani-cl-madrid) [inline src/custom-elements/src/components/drawer/drawer.tsx]: Please, update the value only if content is different to reduce unwanted re-rendering.

Comment (dani-cl-madrid) [reply src/custom-elements/src/components/drawer/drawer.tsx]: Please, don't forget to remove any listener attached. I.e. don't forget to call [observer.disconnect](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver/disconnect) on [componentDidUnload()](https://stenciljs.com/docs/component-lifecycle). Optionally, a better approach could be to do the `observe` and `disconnect` in the [connected](https://stenciljs.com/docs/component-lifecycle#connectedcallback-) and [disconnected](https://stenciljs.com/docs/component-lifecycle#disconnectedcallback-) callbacks.

File: cypress/integration/chi-custom-elements/drawer.spec.js [hunk 2]
```diff
@@ -55,6 +54,19 @@ describe('Drawer', function() {
 
   });
 
+  it('Should have a title if attribute title="" is provided ', function() {
+
+    cy.get('[data-cy="test-top-with-title"]')
+      .scrollIntoView()
```
Comment (dani-cl-madrid) [inline cypress/integration/chi-custom-elements/drawer.spec.js]: ScrollIntoView might be unnecessary as Cypress automatically do this with default configuration.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://eef2114ff034c48be48a0a559fed5322d367d452-chi-pr.enterprise.ctl.io" target="_blank"> https://eef2114ff034c48be48a0a559fed5322d367d452-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/746/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://ddb0f3241aa3ba4c059ca6e5acc5852bc26f1c0b-chi-pr.enterprise.ctl.io" target="_blank"> https://ddb0f3241aa3ba4c059ca6e5acc5852bc26f1c0b-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/747/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://05548758693ce9774744d0ba207017cc59ffb955-chi-pr.enterprise.ctl.io" target="_blank"> https://05548758693ce9774744d0ba207017cc59ffb955-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/748/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-l2s748, https://ops.elasticbox.com/services/instances/i-juamnn, https://ops.elasticbox.com/services/instances/i-jfkgdv, https://ops.elasticbox.com/services/instances/i-sy8olz, https://ops.elasticbox.com/services/instances/i-pvasy5, https://ops.elasticbox.com/services/instances/i-2yp5qh, https://ops.elasticbox.com/services/instances/i-hr4vg5, https://ops.elasticbox.com/services/instances/i-ybpcr8

**Pull request merged**